internal/sdkv2provider/schema_cloudflare_pages_project.go

```diff
@@ -93,6 +93,7 @@ func resourceCloudflarePagesProjectSchema() map[string]*schema.Schema {
 							Elem: &schema.Schema{
 								Type: schema.TypeString,
 							},
+							Default: []string{"*"},
```

Comment:
more about S0008 - https://github.com/bflad/tfproviderlint/tree/main/passes/S008

Reply:
Oh that's fun... issue has been open since 2021 - https://github.com/hashicorp/terraform-plugin-sdk/issues/142

This was defaulted in the API because we wanted to ensure that with Terraform you still had reasonable defaults and didn't accidentally make a project which would never be triggered by Git.

Think we should remove the API default then?

Reply:
i think you can still default it in the API however, the type should be `Computed` to use the API default if none is provided. if that doesn't work, it's probably just better that the API throws an error if this isn't provided (or mark it as `Required` in the schema).